src/MergeContext.js

```diff
@@ -626,8 +629,7 @@ class CommitMessage
 
     // authorField is a {name, value, raw}
     _parseAuthor(authorField) {
-        const trimmedValue = this._trim(authorField.value);
-        const cred = trimmedValue.match(/^([\w][^@<>,]*) <(\S+@\S+\.\S+)>$/);
+        const cred = authorField.value.match(/^([\w][^@<>,]*) <(\S+@\S+\.\S+)>$/);
```

Comment:
If you have not deployed yet, I would relax this a bit to allow tabs and such:

```suggestion
        const cred = authorField.value.match(/^([\w][^@<>,]*)\s<(\S+@\S+\.\S+)>$/);
```

We trim the name value below so we only need to see one space (technically, even that single whitespace character is optional, I guess, but I am not sure we should get _that_ permissive).

Reply:
Done (cd5ad03).